industrial_ci/src/docker.sh

```diff
@@ -100,6 +103,11 @@ function ici_run_cmd_in_docker() {
   return $ret
 }
 
+function docker_cp {
```

Comment:
Am I correct that once the relevant issue in `docker` is resolved this function won't be necessary? If so I'd add a comment here just as a reminder (since I easily forget..), but I leave that up.

Reply:
Not sure if we want to remove later..
But I will add a small comment anyway.